udata/settings.py

```diff
@@ -132,7 +132,7 @@ class Defaults(object):
     SITE_KEYWORDS = ['opendata', 'udata']
     SITE_AUTHOR_URL = None
     SITE_AUTHOR = 'Udata'
-    SITE_GITHUB_URL = 'https://github.com/etalab/udata'
+    SITE_GITHUB_URL = "https://github.com/opendatateam/udata"
```

Comment:
Single quotes are preferred (for now :p)
```suggestion
    SITE_GITHUB_URL = 'https://github.com/opendatateam/udata'
```

Reply:
Fixed